### PR TITLE
Migration of UvvUndulator-ioc to Machine-applications

### PR DESCRIPTION
**Pull request by xresende**
The while gitlab repo `UvvUndulator` has been moved to a folder in `machine-applications` since LNLS will be responsable to maintain, update and deploy this IOC. In principle version control history and commit authorities of incoming files has been preserved.

### CODE REVIEW

File: si-id-epu50/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include si_id_conv/access_rules.as
```
Comment (anacso17) [inline si-id-epu50/MANIFEST.in:1]: typo here si_id_epu50

File: si-id-epu50/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# SI-ID IOC conversion classes and scripts
+
+IOC classes and scripts for insertion devices conversions.
```
Comment (anacso17) [inline si-id-epu50/README.md:3]: # SI-ID EPU50 IOC classes and scripts

IOC classes and scripts for EPU50 insertion devices.

File: si-id-epu50/MANIFEST.in (hunk 1 above)
Comment (xresende) [reply si-id-epu50/MANIFEST.in]: thanks, @anacso17 . the issue is deprecated in new commits

File: si-id-epu50/README.md (hunk 2 above)
Comment (xresende) [reply si-id-epu50/README.md]: deprecated

File: si-id-epu50/si_id_epu50/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"""SI ID Conv package."""
```
[the remark this answers is not in the document]
Comment (xresende) [reply si-id-epu50/si_id_epu50/__init__.py]: deprecated

File: si-id-epu50/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+pyyaml=6.0
+toml=0.10.2
+pydantic=1.10.2
+pcaspy=0.7.3
+pyepics=3.5.0
```
Comment (anacso17) [inline si-id-epu50/requirements.txt:5]: conflict here: we use pyepics 3.4.0...

**Review by fernandohds564 — APPROVED** [review]

**Review by rbatistacardoso — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Pull request merged**